applications/tari_validator_node_web_ui/src/routes/Transactions/TransactionDetails.tsx

```diff
@@ -113,7 +113,7 @@ export default function TransactionDetails() {
 
   useEffect(() => {
     getTransactionByHash();
```

Comment:
Isnt `[]` the default (not sure)? I think the problem is that we dont check if we're already loading or done.

Maybe something like:
```typescript
  useEffect(() => {
    if (!loading && !fee && !logs) {
       getTransactionByHash();
    }
  }, [loading, fee, logs]);
```

Reply:
The default is to run on any change, the `[] ` makes it run only on mount/unmount of the page (which is what we want)

https://stackoverflow.com/questions/53070970/infinite-loop-in-useeffect